init.fish

```diff
@@ -26,11 +26,7 @@ if not set -q TMUX
   set -l tmux_bin (config tmux-zen --get tmux-bin --default tmux)
   set -l session_name (config tmux-zen --get session-name --default local)
 
-  if eval "$tmux_bin has-session -t $session_name 2> /dev/null"
-    exec env -- $tmux_bin new-session -t $session_name \; set destroy-unattached on \; new-window
```

Comment:
I don't know the intent of the `destroy-unattached` part. Maybe it's something worth keeping? Although, when starting a new shell I don't really want to nuke things, if there's some cleaning up to do I think the tmux server is better equipped to perform it.